### DISCUSSION TITLE
Takes 10 seconds to open

### DISCUSSION THREAD

**Discussion by spydercanopus** [iterate-ch/cyberduck#15205] [Q&A]
Like it's waiting on a server response or something.  Fix it and I'll "buy a license".  I appreciate your development.

**Comment by tatze96** [comment]
Please add the following information:
- Which operating system is in use?
- Which version of Cyberduck is in use?
- Do you use the standalone or the App Store version of Cyberduck?

**Reply by spydercanopus** [reply]
Both installations suffering this dreaded waiting game for it to open are:
Win10, 8.6.0, installed from Microsoft Store probably.

I also use a Firewalla to block tracking and ad servers for all my devices.
